vscode-powershell.build.ps1

```diff
@@ -106,7 +100,7 @@ task UpdateReadme -If { $script:IsPreviewExtension } {
     }
 }
 
-task Package UpdateReadme, {
+task Package UpdateReadme, Build, {
```

Comment:
Annoying dependency bug if you've cleaned and then want to package, it should just build.